packages/react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -175,16 +178,7 @@ export class GroupHeaderBase extends React.Component<IGroupHeaderProps, IGroupHe
             </button>
           </div>
 
-          <div
-            className={this._classNames.title}
-            id={this._id}
-            onClick={this._onHeaderClick}
-            role="gridcell"
-            aria-colspan={this.props.ariaColSpan}
-            data-selection-invoke={true}
-          >
-            {onRenderTitle(this.props, this._onRenderTitle)}
-          </div>
+          {onRenderTitle(this.props)}
```

Comment:
>onRenderTitle [](http://example.com/codeflow?start=11&length=13)

If a consumer that passed an `onRenderTitle` relied on the `onHeaderClick` being hooked up for them, won't they now be  broken because it'll call their `onRenderTitle` vs the default one and they'd now have to know that they need to add their own click handler?

Reply:
Got any ideas? Removing `onRenderClick` from the base `div` is part of the accessibility fixes I need to make. I originally did the `onRenderTitle` rework to make it so that handler was preserved.